errbot/backends/slack.py

```diff
@@ -1014,6 +1021,8 @@ def id(self):
             self._id = self._channel.id
         return self._id
 
+    channelid = id
```

Comment:
It doesn't look like this is used anywhere, can you see if it's possible to remove it?